database.js

```diff
@@ -0,0 +1,99 @@
+class Database
+{
+    constructor()
+{
+    this._teamList = [];
+    this.tempId = [];
+    this.id = 0;
+}
+getTeams()
+{
+    return this._teamList;
+}
+getNewId()
```

Comment:
cała ta funkcja - czyste szaleństwo :) Jeśli masz 10 drużyn, to będziesz miał 10 razy sortowanie. Do tego rozwiązujesz problem wyszukiwania największej wartości ....
```
let ids = this._teamList.map(team => team.id)
let max = Math.max(...ids)
```

I do tego jeszcze - przy każdym dodaniu drużyny szukasz na nowo największego maxa. A po co? Jak raz załadujesz drużyny z dynamo, to poszukasz największego maxa i wtedy już nie trzeba przeszukiwac wszystkich drużyn

Reply:
Czyste szaleństwo uporządkowane :))

Reply:
True, poprawione.